$NetBSD: patch-ab,v 1.3 2000/11/29 17:08:08 agc Exp $

Patch from Tom Thai (without this patch, mysql will result in a wrong
date for "select now()")

--- gen/ctime.c	2000/11/29 16:21:13	1.1
+++ gen/ctime.c	2000/11/29 16:23:14
@@ -129,7 +129,7 @@
 ** Prototypes for static functions.
 */
 
-static long			detzcode __P((const char *));
+static int			detzcode __P((const char *));
 static const char *	getnum __P((const char *, int *, int, int));
 static const char *	getsecs __P((const char *, long *));
 static const char *	getoffset __P((const char *, long *));
@@ -175,9 +175,9 @@
 time_t			altzone = 0;
 #endif /* defined ALTZONE */
 
-static long detzcode(const char * codep)
+static int detzcode(const char * codep)
 {
-	long result;
+	int result;
 	int	i;
 
 	result = 0;
